content/docs/alerting/alertmanager.md

```diff
@@ -67,3 +67,8 @@ Silences are configured in the web interface of the Alertmanager.
 The Alertmanager has [special requirements](../clients) for behavior of its
 client. Those are only relevant for advanced use cases where Prometheus
 is not used to send alerts.
+
+## High Availability
+
+Alertmanager supports a mesh configuration to create a cluster for high availability.
```

Comment:
I'd also mention here that all Prometheus servers talk to all Alertmanagers, as that's a common misunderstanding.

Reply:
I do think this is important, could you add this @josedonizetti ?

Reply:
@brancz It's already added. Paragraph below.